x/smartaccount/ante/auth.go

```diff
@@ -81,14 +81,16 @@ func (sad SmartAccountAuthDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simu
 		return ctx, err
 	}
 
-	// Current smartaccount only supports one signer (TODO review in the future)
+	// Current smartaccount only supports one signer
+	// Multiple signers here means that the transaction perform state changes on multiple wallets
```

Comment:
This means that someone can aggregate all the signed messages and broadcast them on the network paying the gas fee on behalf of the users with all the messages being included in the same block?

Reply:
I dont think so. The transaction needs to be signed for the entire set of the messages in a transaction (to be verified). So aggregating different messages signed by different people might not work.